readalongs/api.py

```diff
@@ -0,0 +1,146 @@
+"""
+api.py: API for calling readalongs CLI commands programmatically
+
+In this API, functions take the same arguments as on the readalongs
+command-line interface. The mapping between CLI options and API options is
+that the first long variant of an option described in "readalongs cmd -h" is
```

Comment:
maybe "readalongs <cmd> -h" or some way to specify that cmd is not an actual command?